content/300-guides/050-database/300-seed-database.mdx

```diff
@@ -20,12 +20,40 @@ Watch the following video for an introduction to the `db seed` command:
 
 </TopBlock>
 
-## Seeding your database with TypeScript
+## Requirements for seeding with TypeScript or JavaScript
 
-If you are using TypeScript, follow these steps:
+Prisma's integrated seeding functionality expects the following:
+
+1. A `prisma/seed.ts` or `prisma/seed.js` file.
+2. **One** of the following inside the `seed.js`/`seed.ts` file:
+
+    * A script inside `seed.ts`
+    * An export named `seed` ([2.23.0](https://github.com/prisma/prisma/releases/tag/2.23.0) and later):
+
+      ```ts
+      export const seed = () => {
+        // tests here
+      }  
+      ```
+
+    * A default export ([2.23.0](https://github.com/prisma/prisma/releases/tag/2.23.0) and later):
+
+      ```ts
+      const mytests = () => {
+        // tests here
+      }
+
+      export default mytests;
+      ```
```

Comment:
Using `export` in JavaScript will throw if Node.js target is CommonJS (without a transpiler)
```
SyntaxError: Unexpected token 'export'
```

So we could recommend this for .js

```js
function seed() {
   // Prisma create query to seed models in database
}

module.exports = {
   seed,
}
```

and default
```
function seed() {
   // Prisma create query to seed models in database
}

module.exports = seed
```

Reply:
🙏 Thank you!